client/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM node:16-alpine
-ENV PATH /app/node_modules/.bin:$PATH
+ENV PATH="/app/node_modules/.bin:${PATH}"
```

Comment:
At this point in the process, this path isn't in existance.  It seems with out the ="" bit it will just fail to add to the path if the dir doesn't exist yet.